pkg/network/controllers/vm.go

```diff
@@ -100,13 +90,7 @@ func (v *VMController) Sync(vm *v1.VirtualMachine, vmi *v1.VirtualMachineInstanc
 	vmiCopy.Spec.Domain.Devices.Interfaces = ifaces
 	vmiCopy.Spec.Networks = networks
 
-	hasOrdinalIfaces, err := v.hasOrdinalNetworkInterfaces(vmi)
-	if err != nil {
-		return vm, &syncError{
-			fmt.Errorf("error encountered when trying to check if VMI has interface with ordinal names (e.g.: eth1, eth2..): %v", err),
-			hotPlugNetworkInterfaceErrorReason,
-		}
-	}
+	hasOrdinalIfaces := hasOrdinalSecondaryIfaces(vmi.Spec.Networks, vmi.Status.Interfaces)
```

Comment:
Just to make sure we are covered: With this approach, it may take more cycles to detect the actual naming scheme. Will that have any effect?

Reply:
The hotunplug operation will be blocked until the naming scheme will be known.